resources/contenttweaker/lang/en_us.lang

```diff
@@ -15,8 +15,8 @@ description.crt.consume=§6§o(Will be consumed!)§r
 # CoT new content
 # --------------------------------------
 itemGroup.other=ContentTweaker
-fluid.heavy_metal=Heavy Metal
-fluid.cheese=Cheese
+# fluid.heavy_metal=Heavy Metal
```

Comment:
Why do you keep old commented variants?

Reply:
I just see @ZZZank leaving a lot of them like this too, for no particular reason.

Reply:
Commented lines only make sense when they are NOT commented in other languages.

Reply:
Okay, I will delete it.